dvc/remote/gdrive.py

```diff
@@ -195,10 +207,35 @@ def drive(self):
 
             # Pass non existent settings path to force DEFAULT_SETTINGS loading
             gauth = GoogleAuth(settings_file="")
-            gauth.CommandLineAuth()
 
-            if os.getenv(RemoteGDrive.GDRIVE_USER_CREDENTIALS_DATA):
-                os.remove(self.gdrive_user_credentials_path)
+            try:
+                gauth.CommandLineAuth()
+            except RefreshError as e:
+                raise GDriveAccessTokenRefreshError(
+                    msg="Google Drive's access token " "refreshment is failed",
+                    cause=str(e),
```

Comment:
can we just make it `GDriveAccessTokenRefreshError("blah-blah", e)`?

Reply:
@MaxRis cause shouldn't be stringified, you should pass full exception object to it, it will format it automatically as needed, see any examples in our code.

Reply:
Thanks, have updated as per @efiop suggestion.